Sources/SourceKitLSP/Swift/MacroExpansion.swift

```diff
@@ -252,10 +252,16 @@ extension SwiftLanguageService {
       let expansionURIs = try macroExpansionReferenceDocumentURLs.map {
         return DocumentURI(try $0.url)
       }
+
+      let position = switch macroExpansionReferenceDocumentURLs[0] {
+        case .macroExpansion(let data):
+          data.macroExpansionEditRange.lowerBound
+      }
```

Comment:
Could you add a check that `macroExpansionReferenceDocumentURLs` is not empty so we don’t crash if it is for some reason?